core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -621,30 +621,33 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.SERVER)
           .build();
-  public static final PropertyKey UNDERFS_OBJECT_STORE_READ_RETRY_BASE_SLEEP_MS =
-      new Builder(Name.UNDERFS_OBJECT_STORE_READ_RETRY_BASE_SLEEP_MS)
+  public static final PropertyKey UNDERFS_OBJECT_STORE_RETRY_BASE_SLEEP_MS =
+      new Builder(Name.UNDERFS_OBJECT_STORE_RETRY_BASE_SLEEP_MS)
           .setDefaultValue("50ms")
-          .setDescription("Block reads from an object store automatically retry for transient "
-              + "errors with an exponential backoff. This property determines the base time in the "
-              + "exponential backoff. Only applicable for S3A.")
+          .setDescription("Block reads and gets metadata information from an object store "
```

Comment:
We can simplify to just "Object store operations automatically retry for transient errors ... "